gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStreamIntegrationTest.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.hadoop.fs.gcs;
+
+import com.google.cloud.hadoop.gcsio.GoogleCloudStorageReadOptions;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.UUID;
+
+import static com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystemTestBase.loadConfig;
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.*;
+
+@RunWith(JUnit4.class)
+public class GoogleHadoopFSInputStreamIntegrationTest
+        extends HadoopFileSystemTestBase {
+
+    @ClassRule
+    public static NotInheritableExternalResource storageResource =
```

Comment:
Do not use this initialization, it's deprecated.

Reply:
I used:
`  private static HadoopFileSystemIntegrationHelper ghfsHelper;
  private static FileSystem ghfs;

  @BeforeClass
  public static void setup() throws Throwable {
    GoogleHadoopFileSystemIntegrationTest.storageResource.before();
    ghfsHelper = GoogleHadoopFileSystemIntegrationTest.ghfsHelper;
    ghfs = GoogleHadoopFileSystemIntegrationTest.ghfs;
  }

  @AfterClass
  public static void cleanup() {
    ghfs = null;
    ghfsHelper = null;
    GoogleHadoopFileSystemIntegrationTest.storageResource.after();
  }`

It will be ok?